srml/aura/src/lib.rs

```diff
@@ -51,6 +51,7 @@
 pub use timestamp;
 
 use rstd::{result, prelude::*};
+use parity_codec::Decode;
```

Comment:
It should be added only in `feature = std`, otherwise will trigger a unused code warning.

Reply:
Or just move the use statement into `error_to_string`.

Reply:
@bkchr @tomusdrw done